sonarqube/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,52 @@
+namespace: sonarqube
+app_name: sonarqube
+
+build:
+  NAME: "{{ app_name }}"
+  SOURCE_REPOSITORY_URL: https://github.com/pcarney8/containers-quickstarts.git
+  SOURCE_REPOSITORY_REF: add-sonarqube
+  SOURCE_CONTEXT_DIR: "{{ app_name }}"
+postgresql:
+  POSTGRESQL_DATABASE: sonar
+  VOLUME_CAPACITY: 5
+  POSTGRESQL_PASSWORD: sonar
+  POSTGRESQL_USER: sonar
+deploy:
+  POSTGRES_DATABASE_NAME: "{{ postgresql.POSTGRESQL_DATABASE }}"
+
+
+openshift_cluster_content:
+- object: Environment Setup
+  content:
+  - name: Create Projects
+    template: "https://raw.githubusercontent.com/redhat-cop/cluster-lifecycle/v3.9.0/files/projectrequest/template.yml"
+    action: create
+    params_from_vars:
+      NAMESPACE: "{{ namespace }}"
+      NAMESPACE_DISPLAY_NAME: "{{ namespace }}"
+    tags:
+      - project
+- object: Builds
+  content:
+  - name: sonarqube
+    template: "{{ inventory_dir }}/../../build-docker-generic/.openshift/templates/docker-build-template.yml"
```

Comment:
Could this template be copied under this directory structure instead of referring to something "external"?

Reply:
I'll accept this PR and fix it in my own PR that contains other changes to SonarQube

Reply:
Never mind this comment, my local repo was broken.